samples/Exporters/Console/TestConsoleActivity.cs

```diff
@@ -0,0 +1,47 @@
+﻿// <auto-generated/>
+
+using OpenTelemetry.Trace.Configuration;
+using OpenTelemetry.Exporter.Console;
+using OpenTelemetry.Trace;
+using System.Diagnostics;
+using OpenTelemetry.Trace.Export;
+
+namespace Samples
+{
+    internal class TestConsoleActvity
+    {
+        internal static object Run(ConsoleActivityOptions options)
+        {
+            // Setup exporter
+            var exporterOptions = new ConsoleExporterOptions
+            {
+                Pretty = options.Pretty
+            };
+            var activityExporter = new ConsoleActivityExporter(exporterOptions);
+
+            // Setup processor
+            var activityProcessor = new SimpleActivityProcessor(activityExporter);
+
+            // Enable OpenTelemetry for the "source" named "DemoSource".
+            OpenTelemetrySDK.EnableOpenTelemetry("DemoSource", activityProcessor);
+
+            // Everything above this line is required only in Applications
+            // which decide to use OT.
+
+            // The following is generating activity.
+            // A library would simply write the following line of code.
+            var source = new ActivitySource("DemoSource");
+            using (var parent = source.StartActivity("parent"))
```

Comment:
Is `parent` end up being display name or "enumeration" name?

Reply:
If not a display name, please add code that sets the display name

Reply:
StartActivity("anythinghere") -  "anythinghere" becomes OperationName, DisplayName by default. DisplayName can be overridden, but OperationName cannot be.

I think the intention is DisplayName to become OT SpanName, as OT allows changing it after starting, but Activity.OperationName is fixed at creation.